src/components/ServerDetails/ServerDetails.tsx

```diff
@@ -0,0 +1,119 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import { Details } from '../Details/Details';
+import { Label } from '../Label/Label';
+import { IServerDetailsProps } from './ServerDetails.Props';
+import { Icon } from '../Icon/Icon';
+import { IDiskDetails } from './ServerDetails.Props';
+import './ServerDetails.scss';
+
+export class ServerDetails extends React.PureComponent<IServerDetailsProps, any> {
+
+    constructor(props?: IServerDetailsProps) {
+        super(props);
+    }
+
+    public render() {
+        const counterInfo = 'counter-info';
+
+        const serverStatusClass = classNames({
+            'server-tile-status-ok': this.props.serverStatus === 'OK',
+            'server-tile-status-warning': this.props.serverStatus === 'Warning',
+            'server-tile-status-error': this.props.serverStatus === 'Error'
+        });
+
+        let users = null;
+
+        if (this.props.headerData.numberOfUsers) {
+            users = (<div className={'st-users'} style={{display: 'inline-block', margin: '0px 5px 0px 5px'}}>
+                        <div style={{display: 'inline-block'}}>
+                            <Icon iconName={'icon-User'} style={{padding: '8px 0px 2px 0px'}}></Icon>
+                            <Label style={{display: 'inline-block'}}>{this.props.headerData.numberOfUsers}</Label>
+                        </div>
+                        <div className={'tooltip'}>
+                            <Label className={'tooltip-label'}>{this.props.headerData.numberOfUsers} users online</Label>
+                        </div>
+                    </div>);
+        }
+
+        return (
+            <div className={'server-tile'}>
+                <div className={serverStatusClass}></div>
+                <div className={'header'}>
+                    <div className={'tl-server-name-container'} style={{display: 'inline-block', margin: '0px 5px 0px 0px'}}>
+                        <Label className="server-name">{this.props.headerData.serverName}</Label>
+                        <div className={'tooltip'}>
+                            <Label className={'tooltip-label'}>{this.props.headerData.fqdmServerName}</Label>
+                        </div>
+                    </div>
+                    
+                    <div className={'tl-disk-container'} style={{display: 'inline-block', margin: '0px 5px 0px 5px'}}>
+                        <Icon className="disk" iconName={'icon-LoadWithErrors'} style={{padding: '8px 5px 2px 5px'}}></Icon>
+                        <div className={'tooltip'}>
+                            {this._diskUsageList(this.props.headerData.diskData.disks)}
+                        </div>
+                    </div>
+                    {users}
+                </div>
+                <Details serverType={'Windows 2008 R2'} serverIcon={'icon-World'} serverRoles={
```

Comment:
these serverTyoe and serverRoles props should be passed from the outside. 
Is this server icon class valid?